src/physics/em/detail/SBPositronXsCorrector.i.hh

```diff
@@ -0,0 +1,67 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file SBPositronXsCorrector.i.hh
+//---------------------------------------------------------------------------//
+#include <cmath>
+
+#include "base/Constants.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct with positron data and energy range.
+ */
+SBPositronXsCorrector::SBPositronXsCorrector(Mass               positron_mass,
+                                             const ElementView& el,
+                                             Energy min_gamma_energy,
+                                             Energy inc_energy)
```

Comment:
Looks like clang-format borked this block.

Reply:
Yeah, further indenting that third line pushes it to 81 columns, so it groups the last two together rather than adding a new line. Ugly, but at least we can blame a machine for it and hope that clang-format does better -- or we can try to tweak/change our clang-format parameters to not align stuff (which is good for diffing and good for formatting, but often bad for reading) -- or we can try to manually format each time we see something "ugly" (and then make it uglier by adding clang-format off/on blockss).

Reply:
Nah, I get the clang format quirks. At a glance in github, it seemed like it could fit. :)